src/macos/PolicyWatcher.cc

```diff
@@ -71,7 +71,7 @@ void PolicyWatcher::Execute(const ExecutionProgress &progress)
                                CFArrayCreate(NULL, (const void **)&path, 1, NULL),
                                kFSEventStreamEventIdSinceNow,
                                1.0,
-                               kFSEventStreamCreateFlagFileEvents);
+                               kCFStreamEventNone);
```

Comment:
Reset flags here to the default

https://developer.apple.com/documentation/coreservices/1455376-fseventstreamcreateflags/kfseventstreamcreateflagnone

The existing flag was overkill